### PR TITLE
Moving Account FKs to point to code field

### PR DESCRIPTION
**Pull request by annalee**
The Peace Corps team uses account codes as their primary way of identifying projects. Pointing the Account foreignkeys on Campaign and Project to the 'code' field allows us to pull the code without having to do a table join.

This PR also includes a cleanup to get SirTrevor fields to serialize correctly.

### CODE REVIEW

File: peacecorps/peacecorps/fields.py [hunk 1]
```diff
@@ -56,3 +59,12 @@ def get_prep_value(self, value):
...
         else:   # No GPG; just stick it in the DB
             return plain_text
+
+
+class BraveSirTrevorField(SirTrevorField):
+
+    def value_to_string(self, obj):
+        sirtrev = self._get_val_from_obj(obj)
+        sirjson = json.loads(sirtrev)
+        sirjohn = json.dumps(sirjson)
```
Comment (cmc333333) [inline peacecorps/peacecorps/fields.py]: Would it work if this was replaced with `return str(sirtrev)`? I don't think we need to parse and encode the data.

**Comment by cmc333333** [comment]
This includes some conflicts with migration numbers; `master` is up to 37, but this starts at `36`. Would you mind combining those migrations into a single migration 38? You just need to concatenate all of the `operations` fields.

**Comment by annalee** [comment]
@cmc333333 Django Migrations doesn't actually care about the file names--those are just for our reference. Once of the included migrations is a merge between these migrations and master's, so there shouldn't be a conflict. Is there a particular reason we want to maintain strict sequential numbering?

**Comment by cmc333333** [comment]
My bad, I didn't realize it was so smart!

**Comment by cmc333333** [comment]
Thanks @annalee !

**Comment by annalee** [comment]
You're correct about str() though. I thought I'd tested that and it was escaping things weirdly, but apparently not.

**Pull request merged**